### PR TITLE
Remove Hazelcast from Cluster manager

### PR DESCRIPTION
**Pull request by a5anka**
Following changes were done to remove Hazelcast from cluster manager

-  Hazelcast based network partition listener was removed since it is not required without Hazelcast
- Hazlecast based unique node ID generator was removed since it is not required for a 2 node HA (active-passive) setup
- Removed the Hazelcast map used to detect duplicate node IDs
- Removed Hazelcast socket information from node details

### CODE REVIEW

File: modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterAgent.java [hunk 1]
```diff
@@ -70,20 +61,10 @@
      * @param manager
      *         Cluster manager for the current node
      */
-    void start(ClusterManager manager) throws AndesException;
+    String start(ClusterManager manager) throws AndesException;
```
Comment (abeykoon) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterAgent.java]: What do we return out of this method?

Comment (a5anka) [inline modules/andes-core/broker/src/main/java/org/wso2/andes/server/cluster/ClusterAgent.java]: Current node's registered node ID in the cluster.

**Comment by abeykoon** [comment]
How node IDs are generated/picked up in the new coordination mechanism? Are they removed or are they picked up by config files?

**Comment by a5anka** [comment]
It is a bit similar to what we had before. If a node id is provided in the config that will be used. Otherwise, a unique ID will be generated.

**Pull request merged**